packages/cli/commands/sandbox/delete.js

```diff
@@ -16,8 +27,60 @@ exports.describe = i18n(`${i18nKey}.describe`);
 exports.handler = async options => {
   await loadAndValidateOptions(options);
 
-  logger.log('');
-  logger.log(i18n(`${i18nKey}.temporaryMessage`));
+  const { account } = options;
+  const config = getConfig();
+
+  let accountPrompt;
+  if (!account) {
+    accountPrompt = await deleteSandboxPrompt(config);
+  }
+  const sandboxAccountId = getAccountId({
+    account: account || accountPrompt.account,
+  });
+
+  trackCommandUsage('sandbox-delete', {}, sandboxAccountId);
+
+  let parentAccountId;
+  for (const portal of config.portals) {
+    if (portal.portalId === sandboxAccountId) {
+      if (portal.parentAccountId) {
+        parentAccountId = portal.parentAccountId;
+      } else {
+        const parentAccountPrompt = await deleteSandboxPrompt(config, true);
+        parentAccountId = getAccountId({
+          account: parentAccountPrompt.account,
+        });
+      }
+    }
+  }
+
+  logger.debug(
+    i18n(`${i18nKey}.debug.deleting`, {
+      account: account || accountPrompt.account,
+    })
+  );
+
+  try {
+    await deleteSandbox(parentAccountId, sandboxAccountId);
```

Comment:
Should we add a confirmation here? The ramifications of fat-fingering the wrong account here seem pretty bad.

Reply:
Ah yes, totally forgot about the confirmation step. Good call out!